packages/js/data/src/product-variations/types.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { DispatchFromMap } from '@automattic/data-stores';
+
+/**
+ * Internal dependencies
+ */
+import { CrudActions, CrudSelectors } from '../crud/types';
+import { Product, ProductQuery, ReadOnlyProperties } from '../products/types';
+
+type ProductVariation = Omit< Product, 'name' | 'slug' >;
```

Comment:
If this is the type managed by the datastore then we should exporte it to be used outside right?

Reply:
Great point!  Added in https://github.com/woocommerce/woocommerce/pull/35838/commits/79c1f13dfc5e14750a996d763897eb71e7a99b88